### PR TITLE
Change `id` -> `event_id` and include push section

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

File: api/push-gateway/push_notifier.yaml [hunk 1]
```diff
@@ -17,7 +17,17 @@ paths:
       summary: Notify a push gateway about an event.
       description: |-
         This endpoint is invoked by HTTP pushers to notify a push gateway about
-        an event.
+        an event or update the number of unread notifications a user has.
+        In the former case it will contain selected information about the event.
+        In either case it may contain numeric counts of the number of unread
+        events of different types the user has. The counts may be sent along
+        with a notification about an event or by themselves.
+        Notifications about a particular event will normally cause the user to be
```
Comment (richvdh) [inline api/push-gateway/push_notifier.yaml]: could maybe do with a paragraph break in here?

**Comment by richvdh** [comment]
I think `notification`, `counts` and `devices.items` each need a `title` to make the generated spec right.

**Comment by richvdh** [comment]
By the way, could you try to give your commit messages short [summary lines](http://chris.beams.io/posts/git-commit/#separate) ?

**Comment by dbkr** [comment]
Noticed some other problems too. ptal.

**Comment by richvdh** [comment]
lgtm now.

**Pull request merged**